dialogue-test-common/src/main/java/com/palantir/dialogue/AbstractProxyConfigTest.java

```diff
@@ -62,11 +62,14 @@ public void close() {}
             })
             .build();
 
-    @Rule
-    public final MockWebServer server = new MockWebServer();
+    private final MockWebServer server;
 
-    @Rule
-    public final MockWebServer proxyServer = new MockWebServer();
+    private final MockWebServer proxyServer;
+
+    protected AbstractProxyConfigTest(MockWebServer server, MockWebServer proxyServer) {
```

Comment:
I didn't see any tests out side this repo that extend this class, so while this is technically an API break I think we're fine

Reply:
I don’t believe we publish this module :+1: